src/main/kotlin/net/dungeonhub/application/commands/LookupCommand.kt

```diff
@@ -44,6 +44,21 @@ class LookupCommand : Extension() {
                 }
             }
 
+            publicSubCommand {
+                name = Lookup.Appeal.name
+                description = Lookup.Appeal.description
+
+                action {
+                    respond {
+                        addEmbed {
+                            color(EmbedColor.Default)
+                            title = "Account Review Request"
+                            description = "Please appeal at https://forms.gle/U6reUwtQSwSTvJB47\nPlease make sure your screenshots **are not cropped**!"
```

Comment:
I would phrase it a bit differently, since we have more than one provider
```suggestion
                            description = "Please appeal at once of the following links:\n" + 
                            "- SkyblockZ/Jerry: https://forms.gle/U6reUwtQSwSTvJB47\n" +
                            "Please make sure your screenshots **are not cropped**!"
```